superset/assets/src/explore/components/AdhocFilterEditPopoverSimpleTabContent.jsx

```diff
@@ -0,0 +1,248 @@
+  }
+
+  onComparatorChange(comparator) {
+    this.props.onChange(this.props.adhocFilter.duplicateWith({
+      comparator,
+      expressionType: EXPRESSION_TYPES.SIMPLE,
+    }));
+  }
+
+  refreshComparatorSuggestions() {
+    const datasource = this.props.datasource;
+    const col = this.props.adhocFilter.subject;
+    const having = this.props.adhocFilter.clause === CLAUSES.HAVING;
+
+    if (col && datasource && datasource.filter_select && !having) {
+      if (this.state.activeRequest) {
+        this.state.activeRequest.abort();
+      }
+      this.setState({
+        activeRequest: $.ajax({
+          type: 'GET',
+          url: `/superset/filter/${datasource.type}/${datasource.id}/${col}/`,
+          success: data => this.setState({ suggestions: data, activeRequest: null }),
+        }),
+      });
+    }
+  }
+
+  isOperatorRelevant(operator) {
+    return !(
+      (this.props.datasource.type === 'druid' && TABLE_ONLY_OPERATORS.indexOf(operator) >= 0) ||
+      (this.props.datasource.type === 'table' && DRUID_ONLY_OPERATORS.indexOf(operator) >= 0) ||
+      (
+        this.props.adhocFilter.clause === CLAUSES.HAVING &&
+        HAVING_OPERATORS.indexOf(operator) === -1
+      )
+    );
+  }
+
+  focusComparator(ref) {
+    if (ref) {
+      ref.focus();
+    }
+  }
+
+  render() {
+    const { adhocFilter, options, datasource } = this.props;
+
+    let subjectSelectProps = {
+      value: adhocFilter.subject ? { value: adhocFilter.subject } : undefined,
+      onChange: this.onSubjectChange,
+      optionRenderer: VirtualizedRendererWrap(option => (
+        <FilterDefinitionOption option={option} />
+      )),
+      valueRenderer: option => <span>{option.value}</span>,
+      valueKey: 'filterOptionName',
+      noResultsText: 'No such column found. To filter on a metric, try the Custom SQL tab.',
+    };
+
+    if (datasource.type === 'druid') {
+      subjectSelectProps = {
+        ...subjectSelectProps,
+        placeholder: t('%s column(s) and metric(s)', options.length),
+        options,
+      };
+    } else {
+      // we cannot support simple ad-hoc filters for metrics because we don't know what type
+      // the value should be cast to (without knowing the output type of the aggregate, which
+      // becomes a rather complicated problem)
+      subjectSelectProps = {
+        ...subjectSelectProps,
+        placeholder: adhocFilter.clause === CLAUSES.WHERE ?
+          t('%s column(s)', options.length) :
+          t('To filter on a metric, use Custom SQL tab.'),
+        options: options.filter(option => option.column_name),
+      };
+    }
+
+    const operatorSelectProps = {
+      placeholder: t('%s operators(s)', Object.keys(OPERATORS).length),
+      options: Object.keys(OPERATORS).filter(this.isOperatorRelevant).map((
+        operator => ({ operator })
+      )),
+      value: adhocFilter.operator,
+      onChange: this.onOperatorChange,
+      optionRenderer: VirtualizedRendererWrap((
```

Comment:
so are we just doing readable options for `equals`? any reason not to make them all human readable?

Reply:
(to clarify, I think readable labels are good for the options dropdown, I think the mathematical representation `>`, `!=` etc is still probably better for the label since it needs to be concise.) not set on this though, I would imagine most people would understand either. an alternative I think we discussed would be to include both representations in the dropdown eg `!= (not equal to)`